src/Frontend/Path.hs

```diff
@@ -14,10 +14,11 @@ module Frontend.Path
     , UserPs(..)
     , AdminPs(..)
     , IdeaMode(..)
+    , CommentMode(..)
     , viewIdea, editIdea, commentIdea, createIdea, listIdeas, listTopicIdeas
     , likeIdea, voteIdea, judgeIdea
     , voteCommentIdea, voteCommentIdeaReply, voteCommentWithContext
-    , replyCommentIdea, commentOrReplyIdea, isPostOnly, isBroken
+    , replyCommentIdea, commentOrReplyIdea, isPostOnly, isBroken, onComment
```

Comment:
Your missing the next commits, I preferred using onComment for delete and report instead of export 2 (or 4) more functions.